#request constants
string object
---
#response constants
int8 status